perses/annihilation/new_relative.py

```diff
@@ -70,6 +75,12 @@ def __init__(self, topology_proposal, current_positions, new_positions, use_disp
         self.softcore_alpha=0.5
         self.softcore_beta=12*unit.angstrom**2
```

Comment:
No need to alter this immediately, but something to keep an eye out for in case it causes problems in the future.